api/prisma/schema.prisma

```diff
@@ -0,0 +1,144 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "mongodb"
+  url      = env("MONGOHQ_URL")
+}
+
+type DonationStartDate {
+  date DateTime @map("_date") @db.Date
+  when String   @map("_when")
+}
+
+type UserBadges {
+  /// Could not determine type: the field only had null or empty values in the sample set.
+  coreTeam Json?
+}
+
+type UserCompletedChallenges {
+  completedDate Float
```

Comment:
This is a float???

Reply:
Oh. This is just JavaScript being JavaScript - numbers are either Big or DP Floats

Reply:
I think it's simpler than that. We could have used Date (Mongo supports that after all), but we didn't.